synapse/models/base.py

```diff
@@ -0,0 +1,78 @@
+import logging
+
+import synapse.common as s_common
+import synapse.lib.tufo as s_tufo
+import synapse.lib.module as s_module
+
+logger = logging.getLogger(__name__)
+
+class BaseMod(s_module.CoreModule):
+
+    @staticmethod
+    def getBaseModels():
+        modl = {
+
+            'types': (
+
+                ('source', {
+                    'subof': 'guid',
+                    'doc': 'A unique source of data.'}),
+
+                ('seen', {
+                    'subof': 'comp',
+                    'fields': 'source=source,node=node:ndef',
+                    'doc': 'Annotates the presence of a node from the specififed source.'}),
```

Comment:
Suggested: "Annotates that the data in a node was obtained from or observed by a given source."